Source/WebCore/animation/AcceleratedEffectStackUpdater.cpp

```diff
@@ -58,7 +60,15 @@ void AcceleratedEffectStackUpdater::update()
 
         auto* renderLayer = renderer->layer();
         ASSERT(renderLayer && renderLayer->backing());
-        renderLayer->backing()->updateAcceleratedEffectsAndBaseValues(m_timelines);
+        renderLayer->backing()->updateAcceleratedEffectsAndBaseValues(timelinesInUpdate);
+    }
+
+    for (auto& timeline : timelinesInUpdate) {
+        auto& timelineIdentifier = timeline->identifier();
+        if (!m_timelines.contains(timelineIdentifier)) {
```

Comment:
`contains` followed by `set` is two hash lookups. Consider `add` and checking `addResult`.